### DISCUSSION TITLE
Search in grid after updating from 6.x to 10.x

### DISCUSSION THREAD

**Discussion by Hbitvof** [pimcore/pimcore#12205] [Q&A / Support] [answered]
Hi,

We recently updates Pimcore from 6.x to 10.x (latest version) and we have problems with the search in grid view.

When we want to search on two words we get no results.

In version 6 of Pimcore when we search like "Test Word" we get results. In version 10 of Pimcore when we search like "Test Word" we get zero results. When we search for Test we get results, so the search itself is partly working.

Is this a know issue and is there a fix to get this working again?

**Comment by kingjia90** [comment, marked as answer]
Could you please open a bug in the issues section with more details, if the problem is still reproduceable in the [demo.pimcore.com](demo.pimcore.com)?

In the meantime, could you share which specific version of Pimcore are you using? 10.4.1? There were some fixes recently, so it could really depending on which subversion (at least for what i have changed were applied from [10.3.6](https://github.com/pimcore/pimcore/pulls?q=is%3Apr+search+author%3Akingjia90)+)

**Reply by kingjia90** [reply]
also, please try to run the `bin/console pimcore:search-backend-reindex` command

**Comment by Hbitvof** [comment]
Problem is solved! Thanks for the reply
